### ISSUE TITLE
Problem: When agents have a syntax error, there is no way of knowing which agent

### ISSUE THREAD

**Issue by clacke** [fractalide/fractalide#248]
Solution: Have scheduler-loop catch errors when doing dynamic-require, and reraise them with more data.

**Comment by dmichiels** [comment]
Is this issue still open?
I just try to broke a node, and I get as error message : 
```
Agent load-graph fails with : 
#(struct:exn:fail:read agents/hyperflow/node/model.rkt:54:5: read: unexpected `)' #<continuation-mark-set> (#(struct:srcloc /home/denis/fractalide/fractalide/modules/rkt/rkt-fbp/agents/hyperflow/node/model.rkt 54 5 2174 1)))
```
where the path of the node is clearly written. 

Perhaps a  better error message is what is wanted form this issue?

**Comment by clacke** [comment]
Hmm, that looks sufficient. The issue is too badly written to understand my exact situation when I wrote it, sorry. :-)

**Comment by clacke** [comment]
Now I know what I meant. This was fixed in #320. :-)
